play_yt.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+youtube-dl -o "/tmp/%(id)s.%(ext)s" --extract-audio --audio-format mp3 $1
+play /tmp/$1.mp3
+rm /tmp/$1.mp3
```

Comment:
this might be racey?

Reply:
oic. if two rewards for the same url are redeemed at the same time. we could maybe have a little like... `$extra=$RANDOM` at the start and then save to /tmp/$1$extra, etc? idk, sounds like big effrot to me. up to you jess

Reply:
we might also want to  add `set -u` to the start